tests/models/whisper/test_feature_extraction_whisper.py

```diff
@@ -298,6 +298,7 @@ def test_torch_integration_batch(self):
         )
         # fmt: on
 
+        torch.set_default_device("cuda")
```

Comment:
I verified that this test fails with this addition & without my patch. Is adding this line okay? If not can someone recommend me what is better way to write/update the test case since it depends on cuda?

Reply:
To the best of my knowledge, we are not testing `set_default_device` in Transformers. 
I would rather go with:
```suggestion
        with torch.device("cuda"):
```
followed with indentation:
```python
with torch.device("cuda"):
    input_speech = self._load_datasamples(3)
    feature_extractor = WhisperFeatureExtractor()
    input_features = feature_extractor(input_speech, return_tensors="pt").input_features
```